subcommands/keys/tuf_updates_sign_prod_targets.go

```diff
@@ -0,0 +1,75 @@
+package keys
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+)
+
+func init() {
+	signCmd := &cobra.Command{
+		Use:   "sign-prod-targets --txid=<txid> --keys=<tuf-root-keys.tgz>",
+		Short: "Sign production targets for your Factory with the offline targets key",
+		Long: `Sign production targets for your Factory with the offline targets key.
+New signatures are staged for commit together with TUF root modifications.
+
+There are 3 use cases when this command comes handy:
+- When you added a new offline TUF targets key,
+  and want to sign your Factory production targets with it.
+- When you increase the TUF targets signature threshold,
+  and need to sign your production targets with an additional key.
+- When you remove one of existing offline TUF targets keys,
+  and need to replace its signatures of production targets with signatures by another key.`,
```

Comment:
@kprosise - can you give us a little grammar help?

Reply:
Okay!